kubernetes/Makefile

```diff
@@ -79,6 +108,23 @@ build/vars-merged-with-secrets.yml: var.yml var-$(CLUSTER).yml $(CONCENT_SECRET_
 build/image-version.yml:
 	echo "image_version: $(IMAGE_VERSION)" >> build/image-version.yml
 
+build/control-database-configuration.yml:
+	echo "selected_database:      default"  >> "$@"
+	echo "database_name:         \"{{ control_database_name }}\"" >> "$@"
+	echo "database_user:         \"{{ control_database_user }}\"" >> "$@"
+	echo "database_host:         \"{{ control_database_host }}\"" >> "$@"
+	echo "django_admin_fixtures: true" >> "$@"
+	echo "db_admin_password:     \" {{ control_db_admin_password }}\"" >> "$@"
+	echo "db_user_password:      \" {{ control_db_user_password }}\"" >> "$@"
+
+build/storage-database-configuration.yml:
+	echo "selected_database:   storage" >> "$@"
+	echo "database_name:      \"{{ storage_database_name }}\"" >> "$@"
+	echo "database_user:      \"{{ storage_database_user }}\"" >> "$@"
+	echo "database_host:      \"{{ storage_database_host }}\"" >> "$@"
+	echo "db_admin_password:  \" {{ storage_db_admin_password }}\"" >> "$@"
+	echo "db_user_password:   \" {{ storage_db_user_password }}\"" >> "$@"
```

Comment:
Instead of printing each line separately with `echo` it would be more readable to use the Bash [heredoc](http://tldp.org/LDP/abs/html/here-docs.html) syntax:

``` bash
build/storage-database-configuration.yml:
    cat <<- EOF > "$@"
    selected_database:  storage
    database_name:      "{{ storage_database_name }}"
    database_user:      "{{ storage_database_user }}"
    database_host:      "{{ storage_database_host }}"
    db_admin_password:  "{{ storage_db_admin_password }}"
    db_user_password:   "{{ storage_db_user_password }}"
    EOF
```

Reply:
Done.